vendor/modules.txt

```diff
@@ -1,43 +1,21 @@
-# github.com/Azure/azure-sdk-for-go v41.3.0+incompatible
+# github.com/Masterminds/goutils v1.1.1
```

Comment:
Are you playing Mastermind at the office through the bridge?

Reply:
Good catch, it should not be there. Corrected